mongo-mongo-express/mongoCreds.env

```diff
@@ -0,0 +1,3 @@
+MONGO_INITDB_ROOT_USERNAME=root
+MONGO_INITDB_ROOT_PASSWORD=root
+MONGO_INITDB_DATABASE=mongodb
```

Comment:
We should skip specifying `MONGO_INITDB_DATABASE` as no entrypoint scripts are mentioned.

Ref: https://hub.docker.com/_/mongo